{
  "success": false,
  "return_code": 1,
  "stdout": "",
  "stderr": "Traceback (most recent call last):\n  File \"/Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_v3/10_23_2025/backtests_final/FractalCascade_BTFinal_v1.py\", line 172, in <module>\n    stats = bt.run()\n  File \"/opt/anaconda3/envs/tflow/lib/python3.10/site-packages/backtesting/backtesting.py\", line 1170, in run\n    strategy.next()\n  File \"/Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_v3/10_23_2025/backtests_final/FractalCascade_BTFinal_v1.py\", line 75, in next\n    risk_amount = risk_per_trade * self._broker.starting_cash\nAttributeError: '_Broker' object has no attribute 'starting_cash'\n\nERROR conda.cli.main_run:execute(124): `conda run python /Users/md/Dropbox/dev/github/moon-dev-ai-agents-for-trading/src/data/rbi_v3/10_23_2025/backtests_final/FractalCascade_BTFinal_v1.py` failed. (See above for error)\n",
  "execution_time": 1.325177,
  "timestamp": "2025-10-23T08:58:41.779533"
}